debug-pod.yaml

```
@@ -0,0 +1,18 @@
+spec:
+  hostPID: true
+  hostNetwork: true
+  containers:
+  - name: perf-debug
+    image: verizondigital/kubectl-flame:v0.2.4-perf
+    command: ["/bin/sleep", "3600"]
+    securityContext:
+      privileged: true
+      capabilities:
+        add: ["SYS_ADMIN", "SYS_PTRACE"]
+  hostPID: true
```

YAMLlint (1.37.1)

[error] duplication of key "hostPID" in mapping

(key-duplicates)